circle test failure - settings clients - sessions are listed in clients view

https://circleci.com/gh/mozilla/fxa-content-server/7451

> firefox on any platform - settings clients - sessions are listed in clients view (3.281s)
AssertionError: expected 'Web Session, node-XMLHttpRequest' to equal 'Web Session, node-XMLHttpRequest null'
  at Function.assert.equal  <node_modules/intern/browser_modules/chai/chai.js:2277:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:1582:16>
  at runCallback  <node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/leadfoot/Command.js:543:11>
  at <node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/dojo/Promise.ts:237:7>
  at <node_modules/dojo/nextTick.ts:44:3>
  at _combinedTickCallback  <internal/process/next_tick.js:73:7>
  at process._tickCallback  <internal/process/next_tick.js:104:9>
  at Command.then  <node_modules/leadfoot/Command.js:542:10>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:1581:8>
  at Command.<anonymous>  <tests/functional/lib/helpers.js:55:25>
  at runCallback  <node_modules/leadfoot/Command.js:526:31>
  at Command.<anonymous>  <node_modules/leadfoot/Command.js:543:11>
  at <node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/dojo/Promise.ts:237:7>
  at <node_modules/dojo/nextTick.ts:44:3>
  at _combinedTickCallback  <internal/process/next_tick.js:73:7>
  at process._tickCallback  <internal/process/next_tick.js:104:9>
  at Command.then  <node_modules/leadfoot/Command.js:542:10>
  at Test.sessions are listed in clients view [as test]  <tests/functional/settings_clients.js:80:10>
  at <node_modules/intern/lib/Test.js:191:24>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/browser_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/browser_modules/dojo/nextTick.ts:44:3>
  at _combinedTickCallback  <internal/process/next_tick.js:73:7>
  at process._tickCallback  <internal/process/next_tick.js:104:9>

This fails because @philbooth fixed the borked part on the auth-server in https://github.com/mozilla/fxa-auth-server/pull/1982. The tests need to be updated to be less screwy.

--- a/tests/functional/settings_clients.js
+++ b/tests/functional/settings_clients.js
@@ -79,7 +79,7 @@ define([
         // second session is the node.js session from test setup
         .then(testElementTextEquals(
           '.client-webSession:nth-child(2) .client-name',
-          'Web Session, node-XMLHttpRequest null'
+          'Web Session, node-XMLHttpRequest'
         ))
 
         // clicking disconnect on the second session should update the list

